packages/serve/src/index.ts

```diff
@@ -34,8 +44,18 @@ class ServeCommand {
                 let devServerFlags = [];
 
                 try {
-                    // eslint-disable-next-line
-                    devServerFlags = require('webpack-dev-server/bin/cli-flags').devServer;
+                    devServerFlags = (() => {
+                        // eslint-disable-next-line
+                        const flags = require('webpack-dev-server/bin/cli-flags');
+                        // running the old format of flags
+                        // { devServer: [{...}, {}...] }
+                        if (flags.devServer) {
+                            return flags.devServer;
+                        }
+                        // new flag format
+                        // { flag1: {}, flag2: {} }
+                        return Object.values(flags);
+                    })();
```

Comment:
I think we can simplify it, I will finish :+1:

Reply:
Left it because it would be easier to simplify after - https://github.com/webpack/webpack-cli/pull/2626, as both configs will be objects so easy merge